pennylane/qchem/openfermion_pyscf.py

```diff
@@ -16,14 +16,10 @@
 # pylint: disable=too-many-arguments, too-few-public-methods, too-many-branches, unused-variable
```

Comment:
Will this renaming affect any user-facing interface? 🤔

Reply:
Everything non-private in there is importable at the qchem level:

```
from .openfermion_pyscf import (
    decompose,
    dipole_of,
    meanfield,
    observable,
    one_particle,
    two_particle,
)
```

So, I would say no UI change. Do you have anything particular in mind?

Reply:
Nope, just wanted to know!